act-rules-format.bs

```diff
@@ -245,183 +263,210 @@ For other aspects that are not well defined, an ACT Rule MUST include either a d
+    <li>**Rule 1:** All `img` element _with_ an `alt` attribute</li>
+    <li>**Rule 2:** All `img` element _without_ an `alt` attribute</li>
   </ul>
-
-  <p>The applicability of the composite rule combines the applicability of both atomic rules. This becomes:</p>
+  <p>The applicability of the composite rule combines the applicability of both rules. This becomes:</p>
   <blockquote>
     <p>All `img` elements.</p>
   </blockquote>
 </div>
 
 
 Expectations {#expectations}
-==============================
+----------------------------
+
+An ACT Rule MUST contain one or more expectations. The expectations describe what the requirements are for the [=test targets=] derived from the [applicability](#applicability). An expectation is an assertion about a [=test target=]. This assertion is true when the [=test subject=] satisfies the [=accessibility requirement=]. When a test target meets all expectations, the test target `passed` the rule. If the test target does not meet one or more expectations, the test target `failed` the rule. If there are no test targets, the [=outcome=] for the rule is `inapplicable`.
 
-The expectations describe what the requirements are for the [=test targets=] derived from the [applicability](#applicability).
+Each expectation MUST be distinct, unambiguous, and be written in plain language. 
 
+**Note:** Atomic rules can be used for aggregation, but this isn't the primary use case. The logic for the aggregation would be described in the expectations for the rule.
 
-Expectations for Atomic Rules {#expectations-atomic}
-----------------------------------------------------
 
-An [=atomic rule=] MUST contain one or more expectations. An expectation is an assertion that is true about each [=test target=] (see [Applicability](#applicability)). Each expectation MUST be distinct, unambiguous, and be written in plain language. Unlike in applicability, a certain level of subjectivity is allowed in expectations. Meaning that the expectation has only one possible meaning, but that meaning MAY not fully quantifiable.
+### Expectations for Atomic Rules ### {#expectations-atomic}
 
-When all expectations are true for a test target, the test target `passed` the rule. If one or more expectations are false, the test target `failed` the rule. When atomic rules are used in composite rules, the outcome of the composite rules can be `passed` even if the outcome of one of its constituent atomic rule is `failed`, depending on the [expectations](#expectations-composite) of the composite rule.
+An [=atomic rule=] expectation MUST only use information available in the [input aspects](#input-aspects), from the applicability, and other expectations of the same rule. No other information can be used in the expectation. So for instance, an expectation could be "Expectation 1 is true and ...", but it can't be "Rule XYZ passed and ...". This ensures that atomic rules are encapsulated.
```

Comment:
**Editorial**: This should have an introductory sentence, instead of going directly into requirements about aspects. "Expectations for Composite Rules" does this better.

Reply:
All expectations of an [=atomic rule=] MUST describe the logic that is used to determine a single `passed` or `failed` [=outcome=] for a [=test target=]. The expectation MUST...